.gitmodules

```diff
@@ -1,9 +1,6 @@
 [submodule "third_party/testsuite"]
 	path = third_party/testsuite
 	url = https://github.com/WebAssembly/testsuite
-[submodule "third_party/v8/v8"]
-	path = third_party/v8/v8
-	url = https://chromium.googlesource.com/v8/v8.git
 [submodule "third_party/gtest"]
```

Comment:
I don't fully understand the purpose of `gtest`.
It looks like it is used to build `hexfloat_test`, which I can't find what this is used for.